pkg/storageincentives/agent_test.go

```diff
@@ -92,8 +94,8 @@ func TestAgent(t *testing.T) {
 				incrementBy: tc.incrementBy,
 				block:       tc.blocksPerRound}
 			contract := &mockContract{t: t, baseAddr: addr}
-
-			service := createService(addr, backend, contract, uint64(tc.blocksPerRound), uint64(tc.blocksPerPhase))
+			postageContract := contractMock.New()
+			service := createService(addr, backend, contract, postageContract, uint64(tc.blocksPerRound), uint64(tc.blocksPerPhase))
```

Comment:
If we only expose the `PostageBatchExpirer` interface I mentioned. We dont need to instantiate the contract here.